test/integration/targets/azure_rm_securitygroup/tasks/main.yml

```diff
@@ -85,14 +85,14 @@
       resource_group: "{{ resource_group }}"
       name: mysecgroup
       tags:
-          testing: testing
           delete: never
           baz: bar
+      append_tags: True
   register: output
 
 - assert:
       that:
-          - output.state.tags | length == 3
```

Comment:
For this test, if append_tags is false, this assert should be true? And the append_tags is default to be true. How about test the `false` value here